pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -82,10 +82,14 @@ type ResultsAPIProperties struct {
 	S3AccessKeyID         string `json:"s3_access_key_id,omitempty"`
 	S3SecretAccessKey     string `json:"s3_secret_access_key,omitempty"`
 	S3MultiPartSize       int64  `json:"s3_multi_part_size,omitempty"`
-	LoggingPVCName        string `json:"logging_pvc_name"`
+	LoggingPVCName        string `json:"logging_pvc_name,omitempty"`
+	GcsBucketName         string `json:"gcs_bucket_name,omitempty"`
+	StorageEmulatorHost   string `json:"storage_emulator_host,omitempty"`
 	// name of the secret used to get S3 credentials and
 	// pass it as environment variables to the "tekton-results-api" deployment under "api" container
-	SecretName string `json:"secret_name,omitempty"`
+	SecretName         string `json:"secret_name,omitempty"`
+	GCSCredsSecretName string `json:"gcs_creds_secret_name,omitempty"`
+	GCSCredsSecretKey  string `json:"gcs_creds_secret_key,omitempty"`
```

Comment:
I have added `secret_name` via #1547 to support `S3` credentials. I kept the name too generic to support of other providers too. Can you please check that, possibility of reuse?

Reply:
I did check the possibility of reuse. I determined that we need to add a secret separately for GCS because we mount this secret as a file and add a transformer.  So we need a secret key.